CA-Policy.md

```diff
@@ -28,12 +28,12 @@ As a cloud service provider that is also part of the General Services Agency (GS
 
 The main assessment procedures used are the [Federal Risk and Authorization Management Program (FedRAMP)](https://www.fedramp.gov/). Further, 18F engages an [accredited third-party assessment organization](https://www.a2la.org/accreditation/fedramp) (3PAO) to provide an independent review of the cloud.gov system and organizational operations.
 
-Assessments of 18F operations are performed in tandem with vulnerability scanning, malicious user testing, insider threat assessments, and other tests regularly conducted by the following teams: the cloud.gov team, TTS Infrastructure, GSA Information Security, and a 3PAO. The system is also under continuous monitoring from cloud.gov's Cloud Operations team.
+Assessments of 18F operations are performed in tandem with vulnerability scanning, malicious user testing, insider threat assessments, and other tests regularly conducted by the following teams: the cloud.gov team, TTS Technology Portfolio, GSA Information Security, and a 3PAO. The system is also under continuous monitoring from cloud.gov's Cloud Operations team.
 
 18F takes any results seriously, and it implements remediations as soon as possible. Plans of action and milestones (POAMs) are maintained to ensure any findings are resolved, compensated for, or accepted as an operational requirement.
 
 See CA-2, CA-2(1), CA-2(2), CA-2(3), CA-5, CA-7, CA-7(1), CA-8, CA-8(1).
 
-The Infrastructure Director for the entire Technology Transformation Service serves as the Authorizing Official for cloud.gov.
+The TTS Technology Portfolio Director for the Technology Transformation Service serves as the Authorizing Official for cloud.gov.
```

Comment:
Pretty sure I'm not; might be the CIO. Checking.

Reply:
Thanks for noting this. In fact, the AO for cloud.gov, as a FedRAMP authorized system, is the FedRAMP JAB. What is the correct role to list for you with respect to cloud.gov or login.gov? That you authorize all the external services on which we rely?

Reply:
> you authorize all the external services on which we rely

That's not quite true either; Nate Ciano is the Authorizing Official for SaaS. I'm not sure that my position has an Official™️  role in relation to cloud.gov.

Reply:
```suggestion
```

Reply:
Closing this out: Per @jhoyt22, cloud.gov and other platforms/infrastructure as a service do not have their own Authorizing Officials at GSA. They are ATO'd as part of the systems that inherit from them.

For most SaaS, particularly those that are Enterprise-wide, there's a single "implementation", they get authorized by a single Authorizing Official through GSA's Enterprise Application Services (EAS) / Enterprise Infrastructure
Operations (EIO).